redaxo/src/core/lib/sql/sql.php

```diff
@@ -416,7 +416,7 @@ public function setQuery($query, array $params = [], array $options = [])
 
         try {
             $this->stmt = rex_timer::measure(__METHOD__, static function () use ($pdo, $query) {
-                return $pdo->query($query);
+                return $pdo->query($query) ?: null;
```

Comment:
Hiermit erzeugen wir das nächste Problem, jetzt ist es zwar nicht mehr falsable, aber nullable.
nullable ist in psalm noch nicht aktiviert. Aber sobald ich das angehe, hänge ich dann wieder an dieser stelle, weil unten ggf. rowCount() auf null aufgerufen wird.
Daher wäre mit eine Lösung mit assert oder exception lieber.

Reply:
(Ich kann es auch anpassen)

Reply:
Gerne

Reply:
Es funktioniert auf jeden Fall, dass wir hier nun durch den anderen PR den null-Fehler bekommen 👍